extensions/azurecore/src/azureResource/utils.ts

```diff
@@ -375,11 +376,14 @@ export async function makeHttpRequest(account: azdata.Account, subscription: azu
 		return result;
 	}
 
+	const requestHeaders = {
+		'Content-Type': 'application/json',
+		'Authorization': `Bearer ${securityToken.token}`,
+		...customHeaders
```

Comment:
Is this handling undefined customHeaders properly?

Reply:
Yes. Nothing gets added if the customHeaders is undefined.

Reply:
Ah, guess that happens when there's other properties in the object. Just doing `...undefined` by itself throws an error.

I think it'd be better to still give this a default of an empty object though

`customHeaders: ... = { }`